MarsRover.Tests/ControllerShould.cs

```diff
@@ -6,67 +6,84 @@ namespace MarsRover.Tests
 {
     public class ControllerShould
     {
+        Controller _controller;

```

Comment:
are you missing access modifiers here? e.g. `private readonly Controller _controller;`

Reply:
I thought that if there is no modifier, plus (being a field) the name began with an underscore, it was private? To the best of my knowledge, I have followed this convention throughout this kata (& probably the last few).